tooling/components/io.siddhi.distribution.editor.core/src/main/java/io/siddhi/distribution/editor/core/internal/ExportUtils.java

```diff
@@ -126,13 +124,18 @@ public File createZipFile() throws DockerGenerationException, KubernetesGenerati
         boolean bundlesAdded = false;
         boolean configChanged = false;
         boolean envChanged = false;
-
+        String zipFileName = "siddhi-docker.zip";
+        String zipFileRoot = "siddhi-docker/";
+        if (exportType != null && exportType.equals(EXPORT_TYPE_KUBERNETES)) {
```

Comment:
This could be written as EXPORT_TYPE_KUBERNETES.equals(exportType)

Reply:
Fixed in https://github.com/siddhi-io/distribution/pull/794/commits/299f71bb802e9cd7c0b16f595f2247026f0b1739